polyellipsoid/simulate.py

```diff
@@ -225,9 +251,9 @@ def anneal(
             self.sim.state.thermalize_particle_momenta(
                     filter=self.all, kT=kT
             )
-            self.sim.run(schedule[kT])
+            self.sim.run(schedule[kT], write_at_start=write_at_start)
 
-    def _hoomd_writers(self, group, forcefields):
+    def _hoomd_writers(self, group, forces):
```

Comment:
Looks like `_hoomd_writers` is an instance method that uses only instance attributes. So we can safely remove the arguments from method definition. 
```suggestion
    def _hoomd_writers(self):
```

Reply:
Good catch!